src/xml.ts

```diff
@@ -129,6 +130,17 @@ export const signXml = (
   if (options.signatureAlgorithm != null) {
     sig.signatureAlgorithm = algorithms.getSigningAlgorithm(options.signatureAlgorithm);
   }
+  if (options.signingCert != null) {
```

Comment:
I don't see how in this code cert rotation is supported.

Reply:
Actually in my change, we don't need array or certificate rotation, as we need to add latest certificate in AuthNRequest, but same certificate used in generate service provider metadata function, so I'm also following the same.. Should we reaname this parameter with different name? But that will be duplication then..